libgobuster/statehelpers.go

```diff
@@ -78,23 +109,23 @@ func ValidateState(
 			errorList = multierror.Append(errorList, fmt.Errorf("[!] Wordlist (-w): File does not exist: %s", s.Wordlist))
 		}
 	} else if s.Wordlist != "" {
-		errorList = multierror.Append(errorList, fmt.Errorf("[!] Wordlist (-w) specified with pipe from stdin. Can't have both!"))
+		errorList = multierror.Append(errorList, fmt.Errorf("[!] Wordlist (-w) specified with pipe from stdin. Can't have both"))
 	}
 
-	if s.Url == "" {
-		errorList = multierror.Append(errorList, fmt.Errorf("[!] Url/Domain (-u): Must be specified"))
+	if s.URL == "" {
+		errorList = multierror.Append(errorList, fmt.Errorf("[!] URL/Domain (-u): Must be specified"))
 	}
 
 	if s.Mode == "dir" {
-		if err := ValidateDirModeState(s, extensions, codes, proxy, errorList); err.ErrorOrNil() != nil {
+		if err := validateDirModeState(s, extensions, codes, proxy, errorList); err.ErrorOrNil() != nil {
 			errorList = err
 		}
 	}
 
 	return errorList
 }
 
-func ValidateDirModeState(
+func validateDirModeState(
```

Comment:
Why was this switched to a lowercase?

Reply:
lowercase because it is static context (not meant to be used from another file) per go standards (if I understand correctly)